src/main/java/com/sleekbyte/tailor/utils/ArgumentParser.java

```diff
@@ -67,6 +67,15 @@ public CommandLine parseCommandLine(String[] args) throws ParseException {
     }
 
     /**
+     * Exception thrown when option parsing fails.
+     */
+    public static class ArgumentParserException extends Exception {
```

Comment:
Can you move this exception declaration after the member variables in this file.